man/docker-create.1.md

```diff
@@ -240,6 +242,22 @@ This value should always larger than **-m**, so you should always use this with
 **-w**, **--workdir**=""
    Working directory inside the container
 
+# EXAMPLES
```

Comment:
This is a single example so the subhead is not needed. I may be wrong but I think the unexpected behavior **Is* the fact that signals are silently discarded.   So, rewrite to take this into account.

--->https://gist.github.com/moxiegirl/146407e87e18f92a7880

# EXAMPLE

This example runs a container with the `--interactive` flag and without attaching to STDIN. When STDIN is not attached,  it can result in unexpected behavior: 

    # docker run -a stderr -a stdout -i --sig-proxy=true ubuntu bash

Specifically, any tty-generated signals (CTRL-c, etc) are silently discarded.  If, 
however, `--sig-proxy=false` is set, tty signals are generated and the client 
responds &mdash; not the container.  For this reason, unless you have a use for 
running `-i` without STDIN attached, you should run `-i` with 
`-a stdin` or omit `-a` to attach to STDIN by default.